wrangler-core/src/main/java/io/cdap/wrangler/parser/MigrateToV2.java

```diff
@@ -126,11 +126,13 @@ public String migrate() throws DirectiveParseException {
         }
         break;
 
-        //set-type <column> <type>
+        //set-type <column> <type> [<scale> <round> <rounding-mode>]
```

Comment:
Adjust docs to the latest revision

Reply:
Done, changed the necessary readme files too.